scheduling/priority_scheduling.py

```diff
@@ -0,0 +1,122 @@
+"""
+Implementation of Priority Scheduling
+
+https://en.wikipedia.org/wiki/Scheduling_(computing)#Priority_scheduling
+"""
+
+from operator import attrgetter
+
+"""
+
+Creating a class process which will have
+all required attributes
+
+"""
+
+
+class process:
+    def __init__(self, process_name, arrival_time, burst_time, priority):
+        self.process_name = process_name
+        self.arrival_time = arrival_time
+        self.burst_time = burst_time
+        self.priority = priority
+        self.waiting_time = None
+        self.turn_around_time = None
+
+
+n = int(input("Enter number of processes: ").strip())
+processes = []
+
+
+for i in range(n):
+    process_name = input("Enter process name: ").strip()
+    arrival_time = int(input("Enter arrival time: ").strip())
+    burst_time = int(input("Enter burst time: ").strip())
+    priority = int(input("Enter Priority: ").strip())
+
+    processes.append(process(process_name, arrival_time, burst_time, priority))
+
+"""
+Executed processes from the original array will be deleted
+and will be appended to the array below
+"""
+executed_processes = []
+
+
+"""
+Linear Runtime of the Scheduling
+"""
+current_time = 0
+
+"""
+As long as there exists atleast one element in the original array,
```

Comment:
```suggestion
As long as there exists at least one element in the original array,
```
This will make the `codespell` test pass.